src/Microsoft.Data.SqlClient/src/Microsoft/Data/SqlClient/AssemblyCache.netfx.cs

```diff
@@ -0,0 +1,44 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+#if NETFRAMEWORK
+
+using System;
+using System.Diagnostics;
+using Microsoft.Data.SqlClient.Server;
+
+namespace Microsoft.Data.SqlClient
+{
+    internal static class AssemblyCache
+    {
+        internal static int GetLength(object inst)
+        {
+            // Caller should have allocated enough, based on MaxByteSize
+            return SerializationHelperSql9.SizeInBytes(inst);
+        }
+
+        internal static SqlUdtInfo GetInfoFromType(Type t)
```

Comment:
This method is present in netcore's SqlConnection class, [here](https://github.com/dotnet/SqlClient/blob/main/src/Microsoft.Data.SqlClient/netcore/src/Microsoft/Data/SqlClient/SqlConnection.cs#L2578-L2597). Not sure how you want to tackle that - either to remove the method from netcore's SqlConnection and include AssemblyCache in both projects, or add it to netfx's SqlConnection and remove AssemblyCache from both projects.